MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -346,8 +346,8 @@ public string GetInputArgument(IReadOnlyList<string> inputFiles, MediaProtocol p
             CancellationToken cancellationToken)
         {
             var args = extractChapters
-                ? "{0} -i {1} -threads 0 -v warning -print_format json -show_streams -show_chapters -show_format"
-                : "{0} -i {1} -threads 0 -v warning -print_format json -show_streams -show_format";
+                ? "{0} -i \"{1}\" -threads 0 -v warning -print_format json -show_streams -show_chapters -show_format"
```

Comment:
Why use double quotes? I'm not a backend dev (so this is not blocking ofc) but single quotes seems cleaner imo.

Reply:
From what I can tell windows doesn't like single quoting objects (thanks windows)

This should be tested with a file that has quotes in it's name.

Reply:
To be honest: I don't know - I've just copied what other parts of the code are doing.
Looks like the string is splited by the runtime, not by the OS (it's not a call to bash or some other interpreter) - and only double quotes are supported (see https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.arguments?view=netcore-3.1#remarks).

IMO, it would be much more safe to use an argument list instead of a string - but that would be a much bigger change.